### ISSUE TITLE
Komodo v10 UI: No obvious way to close bottom panel

### ISSUE THREAD

**Issue by we3geeks** [Komodo/KomodoEdit#1347]
### Short Summary
There's no obvious UI control to close the bottom panel.

### Steps to Reproduce
Do something that opens the bottom panel (e.g., find in files). The bottom panel opens but there's no obvious control available to close the panel when you're done with it.

### Expected results
There should be a control enabling the user to close the panel.

### Actual results
There isn't...

### Platform Information
Komodo IDE, version 10.0.0-alpha1, build 88986, platform macosx.
Built on Wed Apr 27 01:55:56 2016.

### Additional Information
![k10-20160502c](https://cloud.githubusercontent.com/assets/2390335/14953804/443118c8-102a-11e6-849a-567e58c164c6.png)

**Comment by lagbox** [comment]
So something in addition to the panel buttons in the upper right of the IDE?

**Comment by Defman21** [comment]
An "X" would be much better to have.

**Comment by mitchell-as** [comment]
@Naatan would be the authority on this one. It's your call @Naatan .

**Comment by we3geeks** [comment]
See attached screenshot from Komodo 9; I don't see anything in Komodo v10 to the controls I've highlighted in this screenshot:
![k9-20160502a](https://cloud.githubusercontent.com/assets/2390335/14957525/0274a4d6-1043-11e6-8626-e6fa38c7d88c.png)

**Comment by Naatan** [comment]
The intended way is to use the toggle buttons in your scrollbar. I agree a close button makes sense though. I'll see if I don't get overwhelmed with other bugs about adding this in before release.

**Comment by Naatan** [comment]
@cgchoffman might be a good one for you. I believe this is only a matter of adding an icon, the close button and behaviour should already be there in the DOM.

**Comment by cgchoffman** [comment]
I wasn't going to add back that drop down beside the close button but I've actually been missing.  Now that you can have many different find results open I often can't easily get to the utility panes.  Anyone else have a thought on that?  I'd like to add it back.

**Comment by Defman21** [comment]
^ Assign wars

**Comment by Naatan** [comment]
Fyi in addition to @cgchoffman's commit that added the close button back I've ensured it does not show for side panes. The reasoning being that the bottom pane can show up without you having explicitly invoked it (eg. when you run a find query), so having a redundant UI element to close it is helpful. This mostly isn't true for sidepanes, you almost always explicitly invoke them. The close button on the side panes really bothered me as it made the tab navigation needlessly cluttered. 

I've also added an indicator to the toolbar which will highlight the left/right/bottom pane toggler when the relevant pane is open (like it was in 9 and before). This should help convey that these buttons toggle the panes.
